include/SDL3/SDL_stdinc.h

```diff
@@ -59,6 +59,9 @@ void *alloca(unsigned);
 char *alloca();
 # endif
 #endif
+#ifndef HAVE_ALLOCA
+# define HAVE_ALLOCA
+#endif
```

Comment:
Because of the `#ifdef`ery above this, `alloca `is effectively always available.
This also means this can be simplified, but I'm not 100% sure:
https://github.com/libsdl-org/SDL/blob/3abb464f103532a0ef7a092a77c42cab32aa20f4/include/SDL3/SDL_stdinc.h#L382-L388

Reply:
Since this is a public header, maybe those constants should have the SDL prefix? Maybe SDL_HAVE_ALLOCA and SDL_DISABLE_ALLOCA?

Reply:
`HAVE_ALLOCA` is no more.
Since the `#ifdef`ery always defines an `alloca` definition, it didn't make sense.
I'll rename it to `SDL_DISABE_ALLOCA`.